### PR TITLE
Style clash in study selector

### PR DESCRIPTION
**Pull request by alisman**
Legacy styles interfere with new styles of autosuggest dropdown.

https://github.com/cBioPortal/cbioportal/issues/2589

Changes proposed in this pull request:
- a
- b

# Checks
- [ ] Runs on Heroku.
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Google Style Guide](https://github.com/google/styleguide).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [GifGrabber](http://www.gifgrabber.com/).

# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). If you are part of the
cBioPortal organization, notify the approprate team (remove inappropriate):

@cBioPortal/frontend
@cBioPortal/backend
@cBioPortal/devops

If you are not part of the cBioPortal organization look at who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them here:

### CODE REVIEW

File: portal/src/main/webapp/news.jsp [hunk 1]
```diff
@@ -1,82 +1,491 @@
-<%--
```
Comment (adamabeshouse) [inline portal/src/main/webapp/news.jsp]: unrelated to this PR?

File: portal/src/main/webapp/css/global_portal.css [hunk 2]
```diff
@@ -908,15 +908,15 @@ html { min-height: 100%; margin-bottom: 1px; min-width: 100%; margin-right: 1px;
...
-#content ul {
-  margin-top:5px; margin-bottom: 5px;
-  font-family: verdana, arial, sans-serif;
-  font-size: 12px;
-  line-height: 18px;
-  color: #333333;
-  text-align: justify;
-  list-style-type:disc;
-}
+/*#content ul {*/
```
Comment (adamabeshouse) [inline portal/src/main/webapp/css/global_portal.css]: can we just delete this then?

File: portal/src/main/webapp/news.jsp (hunk 1 above)
Comment (alisman) [inline portal/src/main/webapp/news.jsp]: crap.  will fix.

File: portal/src/main/webapp/css/global_portal.css (hunk 2 above)
Comment (alisman) [inline portal/src/main/webapp/css/global_portal.css]: sure.

**Pull request merged**